src/main.ts

```diff
@@ -12,6 +12,8 @@ const client = new Client({
 await client.login(discord.token);
 await new Promise<void>((resolve) => {
   client.on('ready', () => {
+    // eslint-disable-next-line
```

Comment:
🤷‍♂️
<img width="1030" alt="Capture d’écran 2023-09-03 à 14 00 47" src="https://github.com/codinglab-io/discord-bot/assets/17296266/bb0a0f9a-681d-422b-a3e6-a83eeddf15af">

Reply:
(function moved to loadModules bu same issue with eslint)